.github/workflows/publish.yml

```diff
@@ -0,0 +1,42 @@
+name: PyPI Automated publish
+
+on:
+  release:
+    types: [created]
```

Comment:
I don't think we need to use this since the workflow is already triggered by pushing tags.